pkg/errors/v1alpha1/types.go

```diff
@@ -83,6 +83,9 @@ type wrapper struct {
 // Error is implementation of error interface
 func (w *wrapper) Error() string { return w.msg }
 
+// Cause is implementation of causer interface
+func (w *wrapper) Cause() error { return w.error }
```

Comment:
Do we need to implement Cause for other error based structures?

Reply:
I guess we don't need to as not all structures have a `error` type field. At most we can implement for `withSatck`.